### PR TITLE
docimports for flutter_driver

### PR DESCRIPTION
**Pull request by goderbauer**
Part of https://github.com/flutter/flutter/issues/150800

### CODE REVIEW

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

File: packages/flutter_driver/lib/src/driver/frame_request_pending_latency_summarizer.dart [hunk 1]
```diff
@@ -8,15 +8,15 @@ import 'timeline.dart';
 /// Event name for frame request pending timeline events.
 const String kFrameRequestPendingEvent = 'Frame Request Pending';
 
-/// Summarizes [TimelineEvents]s corresponding to [kFrameRequestPendingEvent] events.
+/// Summarizes [TimelineEvent]s corresponding to [kFrameRequestPendingEvent] events.
```
Comment (srawlins) [inline packages/flutter_driver/lib/src/driver/frame_request_pending_latency_summarizer.dart:11]: Love all these finds

**Review by srawlins — APPROVED** [review]

**Comment by goderbauer** [comment]
google testing is failing on:

```
Target of URI doesn't exist. #uri_does_not_exist

/// @docImport 'package:flutter/foundation.dart';
```

So, submitting this has to wait on getting that warning ignored in google3.

**Comment by gspencergoog** [comment]
> google testing is failing on:
> 
> ```
> Target of URI doesn't exist. #uri_does_not_exist
> 
> /// @docImport 'package:flutter/foundation.dart';
> ```
> 
> So, submitting this has to wait on getting that warning ignored in google3.

I think we already rewrite regular imports there when we roll.  Perhaps it should also rewrite `@docImports`?

**Comment by goderbauer** [comment]
@srawlins This is strange. Google testing was in the clear for https://github.com/flutter/flutter/pull/151175. But on this change it still complains about non-existing URIs:

```
Target of URI doesn't exist: 'package:flutter_test/flutter_test.dart'. #uri_does_not_exist_in_doc_import

/// @docImport 'package:flutter_test/flutter_test.dart';
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

It's not clear to me how the docimports added by this change are any different to the ones added in https://github.com/flutter/flutter/pull/151175.

**Comment by srawlins** [comment]
This will require cl/651406900.

**Pull request merged**